### ISSUE TITLE
Auth Wizard not properly handling enter key actions

### ISSUE THREAD

**Issue by jeffpaul** [10up/distributor#584]
**Describe the bug**
From @helen's testing on #368, it appears that there is not expected `enter` key actions on the auth wizard.  This is also an accessibility issue in that you cannot cleaning keyboard navigate the auth wizard process.

**Steps to Reproduce**

**Expected behavior**
Expectation is that pressing the enter key would either (1) move you to the next step in the wizard or (2) submit a form / complete connection creation, but that is not currently happening.  

**Screenshots**

**Environment information**
 - Device: 
 - OS: 
 - Browser and version: 
 - Distributor version: 
 - Theme and version: 
 - Other installed plugin(s) and version(s):

**Additional context**
